java/api/src/main/java/io/ray/api/options/PlacementGroupCreationOptions.java

```diff
@@ -30,24 +29,20 @@ public PlacementGroupCreationOptions(
       throw new IllegalArgumentException(
           "`PlacementStrategy` must be specified when creating a new placement group.");
     }
-    this.global = global;
     this.name = name;
     this.bundles = bundles;
     this.strategy = strategy;
   }
 
   /** The inner class for building PlacementGroupCreationOptions. */
   public static class Builder {
-    private boolean global;
     private String name;
     private List<Map<String, Double>> bundles;
     private PlacementStrategy strategy;
 
     /**
      * Set the name of a named placement group. This named placement group is only accessible from
-     * this job by this name via {@link Ray#getPlacementGroup(java.lang.String)}. If you want to
-     * create a named placement group that is accessible from all jobs, use {@link
-     * Builder#setGlobalName(java.lang.String)} instead.
+     * this job by this name via {@link Ray#getPlacementGroup(java.lang.String)}.
```

Comment:
only accessible from within the namespace of this job?

Reply:
Good catch.